packages/db2/src/connection-manager.ts

```diff
@@ -1,20 +1,19 @@
+import type { Connection, ConnectionOptions } from '@sequelize/core';
+import {
+  AbstractConnectionManager,
+  ConnectionError,
+  ConnectionRefusedError,
+} from '@sequelize/core';
 import type { ConnStr, Database as Db2LibDatabase } from 'ibm_db';
+import * as Db2 from 'ibm_db';
 import assert from 'node:assert';
 import NodeUtil from 'node:util';
-import { ConnectionError, ConnectionRefusedError } from '../../errors/index.js';
-import type { ConnectionOptions } from '../../sequelize.js';
-import type { Connection } from '../abstract/connection-manager';
-import { AbstractConnectionManager } from '../abstract/connection-manager';
-import type { Db2Dialect } from './index.js';
-
-// TODO: once the code has been split into packages, we won't need to lazy load this anymore
-// eslint-disable-next-line @typescript-eslint/consistent-type-imports
-type Lib = typeof import('ibm_db');
+import type { Db2Dialect } from './dialect.js';
 
 export interface Db2Connection extends Connection, Db2LibDatabase {
```

Comment:
`Lib` is a private value in the connection class and is not accessible.

```suggestion
export interface Db2Connection extends Connection, Db2.Database {}
```